### DISCUSSION TITLE
Dashboard Filtering

### DISCUSSION THREAD

**Discussion by ghost** [datalust/seq-tickets#1942] [Help] [answered]
I am setting up dashboards for each of my use cases in a small web application and I would like to be able to filter the dashboard based on the "Environment" property (workstation | development | production) so that I only get results specific to the environment I'm interested in. I'm not sure if/how I can do that currently as the only Dashboard Filter options are All Events (Errors | Exceptions | Warnings)

Is this currently possible to do and if so how? If not, what is the recommendation for restricting your dashboard views to a specific environment so you are not combining dev/staging/prod data together?

**Comment by nblumhardt** [comment, marked as answer]
Hi @wes-mytrailmate; if you create signals for each of your environments (https://docs.datalust.co/docs/signals), you can use these to filter dashboards by clicking the little "funnel" icon beside the dashboard title and choosing one or more of your environment signals.

Let me know if this does the trick!

Best regards,
Nick

**Reply by ghost** [reply]
Yes! That works great. Thank you so much for the reference to the docs and your quick reply.
